src/site/_data/i18n/courses.yml

```diff
@@ -10,3 +10,8 @@ learn_forms:
     en: Learn Forms!
   description:
     en: "[TODO]: ..."
+get_started_with_forms:
+  title:
```

Comment:
These don't need to have a "title" sub-property. Could you just hoist the "en" descriptions up under "get_started_with_forms"?

Reply:
I had already tried that and just checked again to see what the error was and that gives me 'can't read property length of undefined', which made me think it needed the title.

I'm going to merge this anyway as it's behind a flag so it will only be us working with it, so if you want to make a PR that updates the menu feel free to.